services/QuillLMS/spec/models/topic_spec.rb

```diff
@@ -70,4 +70,59 @@
     end
   end
 
+  describe '#activity_count' do
+    context 'when a topic is level 1' do
+      it 'should provide the count of activities directly atttached to the topic' do
+        topic = create(:topic, level: 1)
+        activities = create_list(:activity, 6)
+        activities.each { |a|
```

Comment:
since activities isn't explicitly used (i.e. the activity details aren't important, just that it is created), you could simplify lines 77-80 and just create activities_topics which will automatically create the activity:

`create_list(:activity_topic, topic: topic)`